docs/3.0/resources/whats-new-prefect-3.mdx

```diff
@@ -46,3 +46,7 @@ Workers were first introduced in Prefect 2.0 as next-generation agents, and are
 ## Performance enhancements
 
 Prefect 3.0 doesn't just bring new features; it also delivers significant performance improvements. Users running massively parallel workflows on distributed systems such as Dask and Ray will notice substantial speedups. In some benchmark cases, we've observed up to a 98% reduction in runtime overhead. These performance gains translate directly into faster execution times and more efficient resource utilization for your data pipelines.
+
+## Release notes
+
+See release notes for each releaseed version in the [Gitub repository](https://github.com/PrefectHQ/prefect/releases).
```

Comment:
I almost think this could go in an admonition at the top for maximum visibility, since in a sense it most directly answers the question "What's new?"
```suggestion
See release notes for each released version in the [Gitub repository](https://github.com/PrefectHQ/prefect/releases).
```

Reply:
Agree, as more users are on v3. If most users who arrive on this page at this page are making a major version bump, then I think maybe reading through the main content of this page first is best. The TOC on the right side has this item in it, too, so there's a small bit of visibility. I don't have a strongly held preference.